sources/packages/forms/src/form-definitions/studentdependantsappeal.json

```diff
@@ -895,6 +895,26 @@
           },
           "properties": {},
           "lockKey": true
+        },
+        {
+          "input": true,
+          "tableView": false,
+          "key": "calculatedTaxYear",
+          "label": "calculatedTaxYear",
+          "protected": false,
+          "unique": false,
+          "persistent": true,
+          "type": "hidden",
+          "tags": [],
+          "conditional": {
+            "show": "",
+            "when": null,
+            "eq": ""
+          },
+          "properties": {},
+          "calculateValue": "const [programYear] = data.programYear ? data.programYear.split(\"-\") : [];\nvalue = programYear ? programYear -1: \"\";",
```

Comment:
For my understanding, I see that the `value` here is extracted differently from how it is done in the `sfaa` forms. Is it because the `programYear` value received here is different from that received in the `sfaa` forms?

Reply:
In the SFAA form we already extract the starting year for the program year name. Here we  need to extract the starting year and subtract 1.

Reply:
Sounds good. Thanks.